examples/cpp_hello_world/stage_3/main/BUILD

```diff
@@ -13,4 +13,5 @@ cc_binary(
         ":hello_greet",
         "//lib:hello_time",
     ],
+    bad
```

Comment:
We shouldn't commit breaking changes to the examples

Reply:
Good catch. I just fixed it. That was just my leftover from trying different things.